Core/src/main/resources/lang/messages_en.json

```diff
@@ -161,8 +161,9 @@
   "setup.road_width": "<gold>Road width</gold>",
   "setup.road_height": "<gold>Road height</gold>",
   "setup.road_block": "<gold>Road block</gold>",
-  "setup.wall_filling_block": "<gold>Wall filling block</gold><gray> (Block(s) been put under the plot border down to Y1)</gray>",
+  "setup.wall_filling_block": "<gold>Wall filling block</gold><gray> (Block(s) been put under the plot border down to min gen height + 1)</gray>",
   "setup.wall_height": "<gold>Wall height</gold>",
+  "setup.min_gen_height": "<gold>Minimum height from which to generate (for 1.18+ can be negative).</gold>",
```

Comment:
...can be negative down to -63 or something? Would be good to include it I guess.

Reply:
Well technically the worlds can be larger than that in 1.18, and P2 should support it fine